ui/templates.go

```diff
@@ -88,6 +88,6 @@ func NamedSelectTemplates(name string) *promptui.SelectTemplates {
 		Label:    fmt.Sprintf("%s {{.Name}}: ", IconInitial),
 		Active:   fmt.Sprintf("%s {{ .Name | underline }}", IconSelect),
 		Inactive: "  {{.Name}}",
-		Selected: fmt.Sprintf(`{{ %q | green }} {{ %q: | bold }} {{ .Name }}`, IconGood, name),
+		Selected: fmt.Sprintf(`{{ %q | green }} {{ %q | bold }} {{ .Name }}`, IconGood, name+":"),
```

Comment:
You should be able to do something like:

```suggestion
		Selected: fmt.Sprintf(`{{ %q | green }} {{ %q | bold }}: {{ .Name }}`, IconGood, name),
```